english/happy-robot/happy-robot.md

```diff
@@ -9,3 +9,5 @@ He jumps up, but before he rushes out the door he...
 a) Grabs a [weapon](./robot-help/robot-weapon.md), or
 
 b) Enjoys some [refreshments](./robot-refresh/refreshments.md).
+
+c) Plays some [pictionary](./english/quick_draw/AI_Pictionary.md) with an AI.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.